packages/mobile/src/screens/profile-screen/TracksTab.tsx

```diff
@@ -1,15 +1,29 @@
+import { useCallback } from 'react'
+
 import { tracksActions } from 'audius-client/src/common/store/pages/profile/lineups/tracks/actions'
-import { getProfileTracksLineup } from 'audius-client/src/common/store/pages/profile/selectors'
 
 import { Lineup } from 'app/components/lineup'
-import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+import { useProfile } from 'app/hooks/selectors'
+import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
 
 import { EmptyProfileTile } from './EmptyProfileTile'
-import { getProfile } from './selectors'
+import { useProfileTracksLineup } from './selectors'
 
 export const TracksTab = () => {
-  const { profile } = useSelectorWeb(getProfile)
-  const lineup = useSelectorWeb(getProfileTracksLineup)
+  const profile = useProfile()
+  const lineup = useProfileTracksLineup()
+  const dispatchWeb = useDispatchWeb()
+
+  const loadMore = useCallback(
```

Comment:
Just curious, was this an oversight before to not call loadMore?

Reply:
i added a default `loadMore` implementation in lineup that i thought covered everything, so i removed this after that, but turns out some `fetchLinups` need extra data